parsons/action_kit/action_kit.py

```diff
@@ -464,7 +464,7 @@ def create_generic_action(self, page, email=None, ak_id=None, **kwargs):
                                return_full_json=True,
                                **kwargs)
 
-    def bulk_upload_csv(self, csv_file, import_page, autocreate_user_fields=0):
+    def bulk_upload_csv(self, csv_file, import_page, autocreate_user_fields=0, user_fields_only=0):
```

Comment:
Could you indulge me and change to `autocreate_user_fields=False` and `user_fields_only=False` to keep with the style of the rest of the library?